content/en/blog/2022/2022-01-06-meet-our-contributors-APAC-India-region-01.md

```diff
@@ -0,0 +1,84 @@
+
+**Editor:** [Priyanka Saggu](psaggu.com)
+
+---
+
+Good day, everyone :wave: 
+
+Welcome to the first episode of the APAC edition of the "Meet Our Contributors" blog post series.
+
+
+In this post, we'll meet five amazing folks from the India region who have been actively contributing to the Upstream Kubernetes projects in a variety of ways, as well as being the leaders or maintainers of numerous community initiatives.
+
+:dizzy: *Let's get started without further ado ...*   
+
+
+## [Arsh Sharma](https://github.com/RinkiyaKeDad)
+
+Arsh was recently employed with VMware as a Member of Technical Staff. As a new contributor, he realised that 1:1 mentorship opportunities were quite beneficial in getting him started with the upstream project.
+
+He is presently a CI Signal shadow on the Kubernetes 1.23 release team. He is also contributing to the sig-testing and sig-docs projects, as well as the [cert-manager](https://github.com/cert-manager/infrastructure) tools development work that is being done under the sig-architecture.
+
+To the newcomers, Arsh helps plan their early contributions sustainably.
+
+> "I would encourage folks to contribute in a way that's sustainable. What I mean by that is that it's easy to be very enthusiastic early on and take up more stuff than one can actually handle. This can often lead to burnout in later stages. It's much more sustainable to work on things iteratively."
+
+## [Kunal Kushwaha](https://github.com/kunal-kushwaha)
+
+Kunal Kushwaha is a core member of the Kubernetes marketing council. He is also a CNCF ambassador and one of the founders of the "CNCF Students" program. He also served as a Communications role shadow during the 1.22 release cycle.
+
+At the end of his first year, Kunal began contributing to the [fabric8io kubernetes-client](https://github.com/fabric8io/kubernetes-client) project. He was then selected to work on the same project as part of Google Summer of Code. Kunal mentored people on the same project, first through Google Summer of Code then through Google Code-in. 
+
+As an open-source enthusiast, he believes that diverse participation in the community is beneficial since it introduces new perspectives and opinions and respect for one's peers. He has worked on various open-source projects, and his participation in communities has considerably assisted his development as a developer.
+
+
+> “I believe if you find yourself in a place where you do not know much about the project, that's a good thing because now you can learn while contributing and the community is there to help you. It has helped me a lot in gaining skills, meeting people from around the world and also helping them. You can learn on the go, you don't have to be an expert. Make sure to also check out no code contributions because being a beginner is a skill and you can bring new perspectives to the organisation.”
```

Comment:
No need for quote marks IMO.
```suggestion
> _I believe if you find yourself in a place where you do not know much about the
> project, that's a good thing because now you can learn while contributing and the
> community is there to help you. It has helped me a lot in gaining skills, meeting
> people from around the world and also helping them. You can learn on the go,
> you don't have to be an expert. Make sure to also check out no code contributions
> because being a beginner is a skill and you can bring new perspectives to the
> organisation._
```